hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/volume/TestOMVolumeCreateRequest.java

```diff
@@ -210,6 +210,78 @@ public void testValidateAndUpdateCacheWithVolumeAlreadyExists()
         omMetadataManager.getVolumeKey(volumeName)));
   }
 
+  @Test
```

Comment:
@DaveTeng0 - can we add test case for OMTenantCreateRequest as well ?

Reply:
Yup, added. Thanks for the review!